project.godot

```diff
@@ -476,6 +471,11 @@ d_physics_debug={
 "events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":16777250,"physical_scancode":0,"unicode":0,"echo":false,"script":null)
  ]
 }
+g_eject_engulfed={
+"deadzone": 0.5,
+"events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":72,"physical_scancode":0,"unicode":0,"echo":false,"script":null)
+ ]
+}
```

Comment:
This is unrelated to the rest of the changes but Godot is very adamant on rearranging this. I don't think is will cause problems however.

Reply:
Yeah, this seems to just be reordered (here's hoping that other people's Godot doesn't constantly want it the other way around).